src/appReview/saga.ts

```diff
@@ -0,0 +1,31 @@
+import InAppReview from 'react-native-in-app-review'
+import { call, put, select, spawn, takeLatest } from 'redux-saga/effects'
+import { lastInteractionTimestampSelector } from 'src/appReview/selectors'
+import { setLastInteractionTimestamp } from 'src/appReview/slice'
+import { Actions as SendActions } from 'src/send/actions'
+import { safely } from 'src/utils/safely'
+import { ONE_DAY_IN_MILLIS } from 'src/utils/time'
+
+export function* setAppReview() {
```

Comment:
`requestInAppReview`? `promptAppStoreReview`? `askUserToReviewApp` (lol)?

Reply:
Fixed in 52567a4d9d85377436362a213040ac010ba97813!